mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -211,9 +213,11 @@ def Rock_GeneralGemmParamsAttr : Rock_Attr<"GeneralGemmParams", [RockTuningParam
 
   let extraClassDeclaration = [{
     std::string getPerfConfigStr(){
-      const Twine paramsConcat = Twine(getMPerBlock()) + ","
-      + Twine(getNPerBlock()) + ","
+      const Twine paramsConcat =
+        Twine(getBlockSize()) + ","
+      + Twine(getMPerBlock()) + ","
       + Twine(getNPerBlock()) + ","
+      + Twine(getKPerBlock()) + ","
       + Twine(getMPerThread()) + ","
       + Twine(getNPerThread());
```

Comment:
Please adjust the sequence of this serialization so tuning parameters can be interchangeably used and compared.

https://github.com/ROCmSoftwarePlatform/rocMLIR/blob/7898c00ad5c4aae24add7c276123ba3be01c6eff/mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h#L60-L65

Reply:
Both match with the visit?

Reply:
I misread on this one.